src/components/Timestamp.tsx

```diff
@@ -17,7 +20,8 @@ const Timestamp: React.FC<TimestampProps> = ({ timestamp }) => {
     now.getMonth() == date.getMonth() &&
     now.getDate() == date.getDate()
   ) {
-    string = date.toLocaleTimeString(undefined, formatOptions);
+    const { year, month, day, ...pickFormatOptions } = formatOptions;
+    string = date.toLocaleString(undefined, pickFormatOptions);
```

Comment:
I wonder why the properties, `year`, `month`, `day`, were not excepted and not used, though they have set in the lines.

```typescript
  const formatOptions = {
    hour: '2-digit',
    minute: '2-digit',
    year: '2-digit',
    month: 'numeric',
    day: 'numeric',
}
```

Reply:
I used the spread operator for subtract the `year`, `month`, `days`.

I understood that if the timestamp is today's date, the hour and minute are represented as 2-digit via `toLocalTimeString` on this line.

Reply:
Oh, it was my mistake 😢 What I wonder in the first comment was resolved, thanks! 🙏 

It's another topic, I tried it in my local environment, but I saw below:

![image](https://user-images.githubusercontent.com/26626194/88871937-11722300-d254-11ea-90c0-ace55ccba496.png)

It seems `hour: "2-digit"` doesn't work in Firefox.
I looked the bug and it seems resolved in [this comment](https://bugs.chromium.org/p/chromium/issues/detail?id=527926#c14) for Chrome, but not for Firefox 😢 

![image](https://user-images.githubusercontent.com/26626194/88873371-7a0ecf00-d257-11ea-924d-0a62c7e1b49a.png)

```console
// Chrome
> new Date(2015, 8, 1, 3, 0, 0).toLocaleString(undefined, {hour: '2-digit', minute: '2-digit'})
< "03:00 AM"

// Firefox
> new Date(2015, 8, 1, 3, 0, 0).toLocaleString(undefined, {hour: '2-digit', minute: '2-digit'})
< "3:00 AM"
```

I know there is no way to fix the problem except for fix directly Firefox, but I leave it as a comment. 🗒️